documentation/modules/proc-kafka-connect-mounting-volumes.adoc

```diff
@@ -43,42 +55,48 @@ metadata:
 spec:
   # ...
   config:
-    config.providers: file
-    config.providers.file.class: org.apache.kafka.common.config.provider.FileConfigProvider
+    config.providers: file <1>
+    config.providers.file.class: org.apache.kafka.common.config.provider.FileConfigProvider <2>
   #...
   externalConfiguration:
     volumes:
-      - name: connector-config
+      - name: connector-config <3>
         secret:
-          secretName: mysecret
+          secretName: mysecret <4>
 ----
+<1> The name of the configuration provider. Use a comma-separted list for more than one provider.
```

Comment:
Right now it's only really possible to use the `FileConfigProvider`, but let's keep the 2nd sentence in anticipation of  KIP-632.

Reply:
Thanks for clarifying Tom. I changed to:

<1> The alias for the configuration provider. Use a comma-separated list if you want to add more than one provider.

Let me know if you think we need to add more.